js_modules/dagit/packages/core/src/app/Telemetry.tsx

```diff
@@ -31,9 +31,7 @@ const LOG_TELEMETRY_MUTATION = gql`
       ...PythonErrorFragment
     }
   }
-
-  ${PYTHON_ERROR_FRAGMENT}
```

Comment:
For the new `graphql()` API, fragments should no longer be appended to the literal.